Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,72 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            bool calculateAgain = true;
+
+            while (calculateAgain == true)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4854634)